exp/lighthorizon/http_test.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/stellar/go/exp/lighthorizon/services"
+	"github.com/stellar/go/support/render/problem"
+)
+
+func TestUnknownUrl(t *testing.T) {
+	recorder := httptest.NewRecorder()
+	request, err := http.NewRequest("GET", "/unknown", nil)
+	if err != nil {
+		t.Fatalf("unexpected error %v", err)
+	}
```

Comment:
similar comment here about `require`, in fact you can probably use `require` over `assert` in all of the tests (unless you're sure you want to continue the test in case of an error)
```suggestion
	request, err := http.NewRequest("GET", "/unknown", nil)
	require.NoError(t, err)
```

(`require` will basically `t.Fatal` if the assertion doesn't pass)